lib/DAV/WorkspacePlugin.php

```diff
@@ -96,26 +96,38 @@ public function propFind(PropFind $propFind, INode $node) {
 		}
 
 		// Only return the property for the parent node and ignore it for further in depth nodes
-		if ($propFind->getDepth() === $this->server->getHTTPDepth()) {
+		$propFind->handle(self::WORKSPACE_PROPERTY, function () use ($node) {
```

Comment:
I'm not entirely sure about removing this condition. https://github.com/nextcloud/text/pull/2608#issuecomment-1162715522 sounds like workspace needs to be set for nested folders and we rely on it being set in our new tests for depth=0.

My understanding is that this will only be evaluated anyway if the property is requested. Would like to hear @juliushaertl  opinion on this before a merge though.